spec.md

```diff
@@ -2232,6 +2331,58 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  option (alpha_message) = true;
+
+  enum Condition {
+    // Volume health condition is unknown; If this is the initial
```

Comment:
initial w/ respect to ... plugin startup? something else? i think i understand what's being attempted here - some sort of suggestion w/ respect to state machine transitions - and it should be probably made even more clear in such case.

Reply:
Maybe it's better just to say "Volume health condition is unknown" here and remove any other comments?